### PR TITLE
remove open document count for survey

### PR DESCRIPTION
**Pull request by IanMatthewHuff**
Follow up per @claudiaregio @greazer discussion.

-   [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Comment by IanMatthewHuff** [comment]
Hang on a quick second, I was testing this out, and I think it's using an unhooked API for the execution count...checking now...

File: src/client/datascience/dataScienceSurveyBanner.ts [hunk 1]
```diff
@@ -201,18 +187,6 @@ export class DataScienceSurveyBanner implements IJupyterExtensionBanner, IExtens
         }
     }
 
-    private getOpenNotebookCount(type: BannerType): number {
-        switch (type) {
-            case BannerType.InsidersNotebookSurvey:
-                return this.getPersistentState(InsidersNotebookSurveyStateKeys.OpenNotebookCount);
```
Comment (joyceerhl) [inline src/client/datascience/dataScienceSurveyBanner.ts:207]: Can we delete this enum value from InsidersNotebookSurveyStateKeys?

**Review by joyceerhl — APPROVED** [review]

File: src/client/datascience/dataScienceSurveyBanner.ts (hunk 1 above)
Comment (IanMatthewHuff) [reply src/client/datascience/dataScienceSurveyBanner.ts:207]: @joyceerhl Good call. I'll update. Also I made a tweak to the interface that we are using. It was actually hooked up to an old logger that wasn't even functioning anymore for execution count 😱 .

File: src/client/datascience/dataScienceSurveyBanner.ts [hunk 2]
```diff
@@ -127,16 +122,7 @@ export class DataScienceSurveyBanner implements IJupyterExtensionBanner, IExtens
     }
 
     public async activate() {
```
Comment (IanMatthewHuff) [inline src/client/datascience/dataScienceSurveyBanner.ts:124]: This was hooked up to an invalid api (onKernelStateChange is not hooked up anymore). I'm going to look at the other usage of that API to see if I can remove it entirely.

**Pull request merged**